README.md

```diff
@@ -30,10 +30,14 @@ Add `sort-class-members` to the plugins section of your `.eslintrc` configuratio
 				"order": [
 					"[static-properties]",
 					"[static-methods]",
+					"[publicProperties]",
```

Comment:
Can we make the casing on these match the existing convention?

Reply:
Fixed, thanks!